Purchases/Purchasing/StoreKitAbstractions/SK2StoreProduct.swift

```diff
@@ -80,12 +80,12 @@ internal struct SK2StoreProduct: StoreProductType {
 
     var introductoryDiscount: StoreProductDiscount? {
         self.underlyingSK2Product.subscription?.introductoryOffer
-            .flatMap({ StoreProductDiscount(sk2Discount: $0, currencyCode: self.currencyCode)  })
+            .flatMap { StoreProductDiscount(sk2Discount: $0, currencyCode: self.currencyCode)  }
```

Comment:
There are too many spaces between `self.currencyCode)` and `}` 😄

Reply:
I'm not good at this 😛

Reply:
The linter should have caught it 😄